src/audio-worklet/free-queue/README.md

```diff
@@ -0,0 +1,11 @@
+# Free Queue 
+
+A Lock-Free Ring Buffer Implementation, focused on utility for applications related to Audio Processing with WebAudio API.
+
+## Working
+
+It is a Lock-Free implementation based on Single Producer - Single Consumer Concept.
```

Comment:
In general: check the capitalization rule. Not everything needs to be capitalized in the sentence.